organization_user_group_members.go

```diff
@@ -0,0 +1,65 @@
+// Package aiven provides a client for using the Aiven API.
+package aiven
+
+type (
+	// OrganizationUserGroupMembersHandler is the client which interacts with the Organization Users Group Members API on Aiven.
+	OrganizationUserGroupMembersHandler struct {
+		// client is the API client to use.
+		client *Client
+	}
+
+	// OrganizationUserGroupMemberRequest is request structure for the Organization Users Group Member API on Aiven.
+	OrganizationUserGroupMemberRequest struct {
+		// Operation to perform on the group:
+		Operation string `json:"operation"`
+
+		// List of user ids to apply with the operation with
+		MemberIDs []string `json:"member_ids"`
+	}
+
+	// OrganizationUserGroupMember is response element for the Organization Users Group Member List API on Aiven.
+	OrganizationUserGroupMember struct {
+		UserID           string `json:"user_id"`
+		LastActivityTime string `json:"last_activity_time"`
+		UserInfo         struct {
```

Comment:
can we please have it separated? e.g. as a struct called `OrganizationUserGroupMemberUserInfo`

Reply:
ok